packages/abi-coder/src/Interface.ts

```diff
@@ -99,7 +100,7 @@ export class Interface<TAbi extends JsonAbi = JsonAbi> {
 
     return AbiCoder.encode(this.jsonAbi, configurable.configurableType, value, {
       isRightPadded: true,
-      encoding: this.jsonAbi.encoding,
+      encoding: ENCODING_V0,
```

Comment:
Maybe some reminder for the future?

```suggestion
      // TODO: Review support for configurables in v1 encoding when it becomes available
      encoding: ENCODING_V0,
```

Reply:
I've also referenced it in #1672.